common.tf

```diff
@@ -199,6 +199,8 @@ module "dr2_ingest_step_function" {
     ingest_start_workflow_lambda_name             = local.ingest_start_workflow_lambda_name
     ingest_workflow_monitor_lambda_name           = local.ingest_workflow_monitor_lambda_name
     ingest_asset_reconciler_lambda_name           = local.ingest_asset_reconciler_lambda_name
+    ingest_lock_table_name                        = local.ingest_lock_dynamo_table_name
+    ingest_lock_table_gsi_name                    = local.ingest_lock_table_global_secondary_index_name
```

Comment:
Each ddb table can have up to 20 GSIs, to be future flexible is there a better way to name this var?